src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -76,25 +95,28 @@ export class ArtworkSidebarBidAction extends React.Component<
     context_module: Schema.ContextModule.Sidebar,
     action_type: Schema.ActionType.Click,
   })
-  redirectToLiveBidding(user) {
+  redirectToLiveBidding(me: ArtworkSidebarBidAction_me | null) {
```

Comment:
I know that technically any data can be `null` because we don't do strict type checking, but `me` can def be `null`, so wanted to document here in code.